### PR TITLE
COL-947 Initial front-end analytics on profile page

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-947

A few trial balloons to test Mixpanel behavior.

### CODE REVIEW

File: public/app/dashboard/profileController.js [hunk 1]
```diff
@@ -125,6 +125,22 @@
...
+     * Toggle activity breakdown selection (contributions or impacts).
+     *
+     * @param  {String}         filter          Which portion of the breakdown to display
+     * @return {void}
+     */
+    var toggleBreakdown = $scope.toggleBreakdown = function(filter) {
+      $scope.breakdown.selected = filter;
+
+      // Track toggle.
+      analyticsService.track('Toggle breakdown filter', {
```
Comment (johncrossman) [inline public/app/dashboard/profileController.js]: @paulkerschen Is this event name intentionally vague? Is the context clear in MixPanel based on key names below?

File: public/app/dashboard/profileController.js [hunk 2]
```diff
@@ -125,6 +125,22 @@
...
+        'profile_user': $scope.user.id,
```
Comment (johncrossman) [inline public/app/dashboard/profileController.js]: @paulkerschen As we know, `$scope.user` can be `me` or a different user I'm browsing. Do we see that distinction in MixPanel?

File: public/app/dashboard/profileController.js [hunk 3]
```diff
@@ -240,6 +256,14 @@
...
         'id': user.id
       };
+
+      // Track view of another user's profile
+      if (!$scope.isMyProfile) {
+        analyticsService.track('View user profile', {
+          'profile_user': user.id,
+          'referer': document.referrer
+        });
+      }
```
Comment (johncrossman) [inline public/app/dashboard/profileController.js]: @paulkerschen This answers question above. :)

**Comment by johncrossman** [comment]
I see no reason to pop this trial balloon. One question posted, @paulkerschen

File: public/app/dashboard/profileController.js (hunk 1 above)
Comment (paulkerschen) [inline public/app/dashboard/profileController.js]: @johncrossman For event names, I'm going off the specs in https://docs.google.com/document/d/1WqwvMP2HHvhkKAQsDsoxv4trfLR_OSXu7h90fVHI01Q - which are, yes, a little vague in their descriptions and could probably use a round of tightening and specificity once we're verified the mechanics are working.

**Pull request merged**